wolfssl/wolfcrypt/types.h

```diff
@@ -148,7 +149,10 @@
 
 	/* set up thread local storage if available */
 	#ifdef HAVE_THREAD_LS
-	    #if defined(_MSC_VER)
+        #if defined(INTIME_RTOS)
+            /* Thread local storage not supported */
+            #define THREAD_LS_T
```

Comment:
During development I did have "HAVE_THREAD_LS" defined in the IDE/INTIME-RTOS/user_settings.h, but after realizing thread local isn't supported I removed "HAVE_THREAD_LS" from their user_settings.h. This section of code can be reverted or replaced with a compiler warning.